### ISSUE TITLE
EasyGui 0.98.1 on Python 2.6.6

### ISSUE THREAD

**Issue by emteeoh** [robertlugg/easygui#129]
For reasons beyond my control, I'm still working on Python 2.6 for some servers. The documentation for easygui says it works on Python v2, but when I try to import the module, it comes up with the exception:
File "C:\Users\richard\AppData\Python\Python26\site-packages\easygui\boxes\utils.py", line 24, in <module>    raise Exception("You must run on Python 2.7+ or Python 3.4+")

So did I install something wrong? or does it not really support any python that's older than 2.7?

**Comment by robertlugg** [comment]
yes.  Unfortunately, that is correct.  I decided that new users would be running a more recent version.  May I suggest that you try Anaconda?  Even if you don't have root/install privileges, you can install multiple versions of python (by default they go in your home directory).  You can do this without messing up anything else on your system.

**Comment by geekgamer2001** [comment]
ohh that explains it
